Kwf_js/EyeCandy/Lightbox/Lightbox.js

```diff
@@ -41,11 +44,34 @@ Kwf.onJElementReady('.kwfLightbox', function lightboxEl(el) {
     l.closeHref = window.location.href.substr(0, window.location.href.lastIndexOf('/'));
     l.contentEl = l.innerLightboxEl.find('.kwfLightboxContent');
     l.style.afterCreateLightboxEl();
-    l.style.onShow();
     l.style.onContentReady();
+    l.style.fixContent();
     el[0].kwfLightbox = l;
     Kwf.EyeCandy.Lightbox.currentOpen = l;
 
+    var transformName = Modernizr.prefixed('transform');
```

Comment:
please add comment what this code tries to do and why so complicated
(probably refactor to own method with a "good name")